src/routes/brands/deleteBrand.ts

```diff
@@ -23,14 +23,16 @@ const url = "/brand/:id";
 
 const handler: RouteHandler<DeleteBrandRoute> = async (req, reply) => {
   try {
-    const brandId = req.params.id;
+    const { id: brandId } = req.params;
     const brandToDelete = await BrandModel.query().findById(brandId);
 
     if (!brandToDelete) {
       return reply.status(404).send({
-        error: "Not fount",
-        statusCode: 404,
-        message: "This brand does not exist",
+        error: {
+          error: "Not fount",
+          code: "notFount",
```

Comment:
please capitalize it

Reply:
I just wanted to let you know that this suggestion was not applied. @leandro0day, please make sure always to use all the suggestions.